input/templates/my-account-change-password.hbs

```diff
@@ -18,42 +18,52 @@
           <div class="my-account-content">
             <div class="row">
               <div class="col-sm-3">
-                {{> myaccount/my-account-sidebar positionSeven=true}}
+                {{> myaccount/my-account-sidebar changeMyPasswordTab=true}}
               </div>
               <div id="my-account-desktop-content" class="col-sm-9">
-                <form id="form-change-password" action="{{@root.meta._links.changePassword.href}}" method="POST">
-                  {{> form/global-messages messages=content.changePassword.feedbackMessage}}
-                  <input type="hidden" name="csrfToken" value="{{@root.meta.csrfToken}}"/>
-                  <input type="hidden" name="id" value="{{content.address.id}}"/>
```

Comment:
I have removed the address ID field, since we are not dealing with addresses anymore ;)